vue-app/src/api/bright-id.ts

```diff
@@ -82,3 +87,36 @@ export async function registerUser(
   )
   return transaction
 }
+
+export async function getBrightId(contextId: string): Promise<BrightId> {
+  const brightId: BrightId = {
+    isLinked: false,
+    isSponsored: false,
+    isVerified: false,
+  }
+
+  try {
+    const verification = await getVerification(contextId)
+    brightId.isLinked = true
+    brightId.isSponsored = true
+    brightId.isVerified = !!verification?.unique
```

Comment:
@corwintines this is one of the main changes you need to be worried about if you branch your work off `develop`. I would suggest to start from this branch and merge everything here.